jwthenticator/client.py

```diff
@@ -129,7 +129,7 @@ async def authenticate(self) -> None:
         :return: None or raises exception if fails.
         """
         if self._key is None:
-            raise Exception("Missing key")
+            raise ValueError("Missing key")
```

Comment:
This should be a `TypeError` as this is not a problem with the value but with the type (None)